anchore_engine/common/helpers.py

```diff
@@ -244,6 +244,15 @@ def extract_analyzer_content(image_data, content_type, manifest=None):
                         ret[k] = avalue
             except Exception as err:
                 raise Exception("could not extract/parse content info - exception: " + str(err))
+        elif 'pkgs.{}'.format(content_type) in idata['imagedata']['analysis_report']['package_list']:
+            # catchall for additional pkg types
+            try:
+                adata = idata['imagedata']['analysis_report']['package_list']['pkgs.{}'.format(content_type)]['base']
+                for k in list(adata.keys()):
```

Comment:
agreed, thanks @alfredodeza - I'm going to leave this as it is for this PR, since this function is using the list(dict.keys()) throughout, and I'd prefer if a change were made more uniformly rather than just once in the implementation.